test/model/store_test.dart

```diff
@@ -1191,6 +1194,21 @@ void main() {
         checkLastRequestFcm(token: '456def');
       }
     }));
+
+    test('use provided appId from packageInfo', () => awaitFakeAsync((async) async {
+      final origTargetPlatform = debugDefaultTargetPlatformOverride;
+      addTearDown(() => debugDefaultTargetPlatformOverride = origTargetPlatform);
+      debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
+      addTearDown(testBinding.reset);
+      testBinding.firebaseMessagingInitialToken = '012abc';
+      testBinding.packageInfoResult = eg.packageInfo(packageName: 'com.example.test');
+      addTearDown(NotificationService.debugReset);
+      await NotificationService.instance.start();
+      prepareStore();
```

Comment:
nit: use a blank line to organize the steps, similar to the neighboring tests

```suggestion
      await NotificationService.instance.start();

      prepareStore();
```